### PR TITLE
update imaging docs

### PR DESCRIPTION
**Pull request by rtasson**
This PR updates the imaging docs a bit, particularly dressing up the bits about scraping an image off an AmpliPi with `rpiboot`. It also fixes a bug that gave me hassle this morning.

### CODE REVIEW

File: docs/pi_imaging.md [hunk 1]
```diff
@@ -60,7 +69,7 @@ The following is an example where the End sector was 10158079.
 ```sh
 end=10158079
 bs=512
-sudo dd if=/dev/sdX of=amplipi.img bs=1MiB count=$[($end+1)*$bs/1024/1024] status=progress
+sudo dd if=/dev/sdX of=amplipi.img bs=1MiB count=$[($end*$bs/1024/1024)+1] status=progress
```
[the remark this answers is not in the document]
Comment (Lohrer) [reply docs/pi_imaging.md:72]: This indeed does appear to be a bug. Looks like it was already fixed in the latest imaging script though: [imaging -> image_shrink.bash#131](https://github.com/micro-nova/AmpliPi/blob/3b903b51d4b1e1cdd91414e17aaab2f964e617c2/scripts/image_shrink.bash#L131)

File: docs/pi_imaging.md [hunk 2]
```diff
@@ -198,5 +207,6 @@ To keep the clean state of the image, remove any SSH keys
 and command history.
 ```
 rm ~/.ssh/authorized_keys
+rm ~/amplipi-dev/house.json*
```
Comment (Lohrer) [inline docs/pi_imaging.md:210]: This should be handled by the (cleanup)[https://github.com/micro-nova/AmpliPi/blob/main/scripts/cleanup] script that Mahdee runs at checkout, but can't hurt here either.

File: docs/pi_imaging.md [hunk 3]
```diff
@@ -43,6 +43,15 @@ sudo dd if=/dev/sdaX bs=4MiB | gzip > amplipi.img.gz
...
 While the Pi is mounted with `rpiboot`, use the partition manager of your choice to shrink the root partition as far as possible.
 GParted is a good graphical partitioning tool.
+
+This can also be accomplished through CLI tools:
+```sh
+sudo e2fsck -f /dev/sdaX
+sudo resize2fs -Mp /dev/sdaX # this command will tell you how many blocks it has been shrunk into
+sudo fdisk  /dev/sdaX
+```
+Follow [this guide](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/s2-disk-storage-parted-resize-part), using sector sizes for the end of the partition. Some math is necessary to determine the correct end of the partition using this method. `resize2fs` will print 4k blocks, but `fdisk` provides 512byte sectors.
```
Comment (Lohrer) [inline docs/pi_imaging.md:53]: This seems like a valid addition to these notes, but again see [image_shrink.bash](https://github.com/micro-nova/AmpliPi/blob/3b903b51d4b1e1cdd91414e17aaab2f964e617c2/scripts/image_shrink.bash#L80) for the automated method.

**Review by Lohrer — APPROVED** [review]
These changes look good.

**Comment by rtasson** [comment]
I'll make sure to give that `imaging` branch a whirl and probably PR it next time I build one; thanks for reminding me about it!

**Pull request merged**